index.html

```diff
@@ -119,19 +119,18 @@
 
 @media only screen and (max-device-width: 600px) {
   .pagedjs_page {
-    width: 100%;
+    width: 100%!important;
     height: calc(100 / 23 * 29.7)!important;
     margin: calc(100 / 23 * 1vw) auto!important;
   }
 
   .pagedjs_page_content, .pagedjs_area, .pagedjs_pagebox, .pagedjs_sheet, .content, .pagedjs_margin .pagedjs_margin-bottom-center .hasContent, .pagedjs_margin-content, span, .Normal {
-    width: 100vw!important;
-    text-align: justify!important;
+    width: 90vw!important;
```

Comment:
should we change it back to 100 now? does the old code come from an official sample? if so, would be great to keep our code as similar as possible to make it easier to maintain in the future

Reply:
when I change it back to 100, it covers 100% of the width, so there is no free space from left and right. 
also, I'm not sure what you mean by official sample, did you find more than this? https://www.pagedjs.org/documentation/ unfortunately they don't mention responsivity at all (at least I didn't find it)

Reply:
oh I thought you copied some of the CSS from somewhere? if not that's fine of course